app/Exceptions/Handler.php

```diff
@@ -57,7 +57,7 @@ public function report(Exception $exception)
     public function render($request, Exception $exception)
     {
         if (!$request->is("api/*")) {
-            return parent::render($request, $exception);
+            return parent::report($request, $exception);
```

Comment:
Was there a reason to invoke report? I didn't understand the change...

This line looked fine, it is the render function so the call to the parent method was correct.

Reply:
I didn't realize that each exception called both render and report so I made the change to make sure all necessary exceptions got reported.
I looked at some other examples of exception handling and I think both methods will get called every time so the change was unnecessary